web/client/utils/ogc/Filter/__tests__/fromObject-test.js

```diff
@@ -37,6 +37,10 @@ const COMPARISON_TESTS = [
     {
         cql: "PROP like 'a'",
         expected: '<ogc:PropertyIsLike matchCase="true" wildCard="*" singleChar="." escapeChar="!"><ogc:PropertyName>PROP</ogc:PropertyName><ogc:Literal>a</ogc:Literal></ogc:PropertyIsLike>'
+    },
+    {
+        cql: "PROP between 1 and 3",
+        expected: '<ogc:PropertyIsBetween><ogc:PropertyName>PROP</ogc:PropertyName><ogc:Literal>1</ogc:Literal><ogc:Literal>3</ogc:Literal></ogc:PropertyIsBetween>'
```

Comment:
Is this correct ? I see this syntax 
```
<ogc:PropertyIsBetween>
              <ogc:PropertyName>PERSONS</ogc:PropertyName>
              <ogc:LowerBoundary>
                <ogc:Literal>2000000</ogc:Literal>
              </ogc:LowerBoundary>
              <ogc:UpperBoundary>
                <ogc:Literal>4000000</ogc:Literal>
              </ogc:UpperBoundary>
            </ogc:PropertyIsBetween>
```

Here I can not see the enclosing tags:
```
<ogc:LowerBoundary>
 <ogc:UpperBoundary>
```

Reply:
GeoServer is happy with this syntax, but I can test if it works also with the more verbose one

Reply:
@offtherailz I added the lowerboundary and upperboundary (GeoServer seems still happy, so let's go for the by specification syntax).